tools/memento-cid-differential/run_differential.sh

```
@@ -0,0 +1,77 @@
+TOTAL_FILES=0
+TOTAL_DIVERGED=0
+
+for src in "$CORPUS_DIR"/*.py; do
+  name="$(basename "$src")"
+  TOTAL_FILES=$((TOTAL_FILES + 1))
+  host_out="$OUT_DIR/${name}.host.jsonl"
+  docker_out="$OUT_DIR/${name}.docker.jsonl"
+
+  python3 "$SCRIPT_DIR/memento_walker.py" "$src" > "$host_out"
+
+  rel_src="${src#$REPO_ROOT/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)